Unreal/SystemLocalized/det/UnrealI.det

```diff
@@ -349,11 +349,11 @@ Message7="Drücke Feuer zum Neustarten"
 
 [KingOfTheHill]
 ; EN: ClassCaption="King of the Hill"
-ClassCaption="King of the Hill"
+ClassCaption="König des Hügels"
```

Comment:
I intentionally did not translate this from "King of the Hill", as thats a fairly well known English term and a common game mode in video games. I've yet to see anyone say or write "König des Hügels" anywhere.

Reply:
Reverted the change to the original string.